src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/HandlerFactory.java

```diff
@@ -74,6 +74,8 @@ public HandlerFactory() {
         register(TokenTypes.INDEX_OP, IndexHandler.class);
         register(TokenTypes.LITERAL_SYNCHRONIZED, SynchronizedHandler.class);
         register(TokenTypes.LAMBDA, LambdaHandler.class);
+        register(TokenTypes.ANNOTATION_DEF, ClassDefHandler.class);
+        register(TokenTypes.ANNOTATION_FIELD_DEF, AnnotationDefHandler.class);
```

Comment:
Separate question. For annotation definition you use `ClassDefHandler`.
`AnnotationDefHandler` looks like it coded like `MemberDefHandler`.
We can't use that handler, or give them common parents, instead? Does it make sense to do this?